app_connections/models/comments.model.js

```diff
@@ -18,3 +19,27 @@ exports.fetchComments = (review_id) => {
       return rows;
     });
 };
+
+exports.addComment = (newComment, review_id) => {
+  const { username, body } = newComment;
+  let count = 0;
+  users.forEach((user) => {
+    if (username !== user.username) {
+      count++;
+    }
+  });
+  if (count === users.length) {
+    return Promise.reject({
+      status: 400,
+      msg: "Invalid username",
+    });
+  }
```

Comment:
This is fine but you can use a more elegant way using the users you have, refactor only if you want